templates/DatasetSubmission/file-submit.html.twig

```diff
@@ -31,7 +33,14 @@
             </div> <!-- tabs-1 -->
 
             {% set sftpuser=app.user.isPosix() %}
-            <div id="tabs-2"> <!-- Upload via SFTP/Ingest-->
+            {% if largefile_tab != 'disabled_hidden' %}
+                {% if largefile_tab == 'disabled_message'%}
+                    <div id="tabs-2" class="disabled-tab" style="background-color: lightgray"> <!-- Upload via SFTP/Ingest-->
```

Comment:
remove the inline css

Reply:
Or add to class

Reply:
It doesn't work added to the class. I too don't like using inline, but I couldn't get it to override otherwise.